examples/mnist.py

```diff
@@ -97,10 +97,11 @@ def create_iter_functions(dataset, output_layer,
     batch_slice = slice(
         batch_index * batch_size, (batch_index + 1) * batch_size)
 
-    objective = lasagne.objectives.Objective(output_layer, loss_function=lasagne.objectives.multinomial_nll)
+    objective = lasagne.objectives.Objective(output_layer,
+        loss_function=lasagne.objectives.categorical_crossentropy)
 
     loss_train = objective.get_loss(X_batch, target=y_batch)
-    loss_eval = objective.get_loss(X_batch, target=y_batch, deterministic=True)
+    loss_eval = objective.get_loss(X_batch, target=y_batch,deterministic=True)
```

Comment:
The problem with the examples is that some of them fail with `ImportError` when included for pytest.  So adding the pep8 check would likely come at the expense of making the examples less comprehensible (with a weird `pytest.skiptest` thingy or something).  I'd see we use our eyeballs for the examples; looks like we have some pretty good ones on board. ;-)